crates/turborepo-lockfiles/src/lib.rs

```diff
@@ -27,7 +31,7 @@ pub struct Package {
 // This trait will only be used when migrating the Go lockfile implementations
 // to Rust. Once the migration is complete we will leverage petgraph for doing
 // our graph calculations.
-pub trait Lockfile: Send + Sync {
+pub trait Lockfile: Send + Sync + Any {
```

Comment:
This is a restriction on the trait, but `Any` is already implemented for any `T: 'static` which was already a restriction from our usage of `Box<dyn Lockfile>` throughout the codebase.